src/figuresClasses.ts

```diff
@@ -1,19 +1,96 @@
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 export interface Figure {
-
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape;
+
+  color: Color;
+
+  a: number;
+
+  b: number;
+
+  c: number;
```

Comment:
You have these values in the constructor. Don't need to write it here
```suggestion
  shape: Shape = "triangle;
```